packages/misc/src/lib/misc.ts

```diff
@@ -279,21 +276,16 @@ export const log = (...args: any): void => {
     return;
   }
 
-  if (globalThis?.litConfig?.debug !== true) {
```

Comment:
🤔 Why does removing this check and the one on L310-L312 below not result in logging just being blanket enabled even when debug is set to false?

Reply:
If you look in `core` where the logger is [created](https://github.com/LIT-Protocol/js-sdk/blob/feature/lit-3911-add-request-id-aggregation-to-logger-when-logging-is-diabled/packages/core/src/lib/lit-core.ts#L208) we now will set the logger to `OFF` which will prevent logs from outputting as it won't have a handler, nor will logs be stored. But it will allow a new logger child to be created which will aggregate request ids.